src/app/pages/component-category-list/_component-category-list-theme.scss

```diff
@@ -8,4 +8,9 @@
   .docs-component-category-list-card-summary {
     color: mat-color($foreground, secondary-text);
   }
+
+  .mat-card {
+    box-shadow: none !important;
```

Comment:
We shouldn't use `!important` (also card is pretty much just a div with `box-shadow` so now that we're turning off the `box-shadow` does it make sense to just use a div instead?

Reply:
I changed it to a div. Also I feel that once we get the pictures in it'll make more sense for them to be in divs